platform/android/app/src/main/kotlin/coop/polypoly/polypod/Authentication.kt

```diff
@@ -15,36 +15,55 @@ class Authentication {
 
         fun shouldShowBiometricsPrompt(context: Context): Boolean {
             return biometricsAvailable(context) &&
-                Preferences.getBiometricCheck(context) &&
-                !Preferences.getBiometricEnabled(context) &&
+                Preferences.isBiometricCheck(context) &&
+                !Preferences.isBiometricEnabled(context) &&
                 !Preferences.isFirstRun(context)
         }
 
         fun setUp(
             activity: FragmentActivity,
-            setupComplete: () -> Unit
+            newStatus: Boolean,
+            setupComplete: (Boolean) -> Boolean
```

Comment:
What is the return value of the callback used for? It does not seem to be used internally.